src/LinuxTracing/UprobesUnwindingVisitorTest.cpp

```diff
@@ -127,31 +162,15 @@ TEST_F(UprobesUnwindingVisitorTest, VisitValidStackSampleWithoutUprobesEmitsEven
   };
   event.ring_buffer_record->sample_id = sample_id;
 
+  EXPECT_CALL(return_address_manager_, PatchSample).WillRepeatedly(Return());
```

Comment:
What is this for? Is this actually required for behavior or is it for the "uninteresting calls" warnings?

Reply:
That is for the warning only.

Reply:
I think that `.Times(AnyNumber())` is clearer then (if it works, haven't tried it).

Reply:
Or just .Times(1)? as I expect only one call?

Reply:
Probably even better then.